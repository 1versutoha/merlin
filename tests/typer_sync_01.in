["reset","name","test.ml"]
["tell","struct","let f () = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\nlet _ = ()\n    \nlet _ = f 5\n"]
["tell","end",null]
["seek","exact",{"line":19,"col":0}]
["refresh"]
["seek","exact",{"line":19,"col":0}]
["errors"]
