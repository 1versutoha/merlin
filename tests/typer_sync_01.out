["return",{"line":1,"col":0}]
["return",false]
["return",true]
["return",{"line":19,"col":11}]
["return",true]
["return",{"line":19,"col":11}]
["return",[{"type":"type","start":{"line":19,"col":10},"end":{"line":19,"col":11},"valid":true,"message":"This expression has type int but an expression was expected of type unit"}]]
